includes/class-newspack-newsletters.php

```diff
@@ -86,6 +88,23 @@ public function __construct() {
 		}
 	}
 
+	/**
+	 * Filter search results, disallowing non-public newsletter posts.
+	 *
+	 * @param array $posts The found posts.
+	 */
+	public static function filter_search( $posts ) {
```

Comment:
```suggestion
	public static function filter_search( $posts ) {
		if ( is_admin() ) {
			return $posts;
		}
```
Would this do the trick?

Reply:
I've added that and some more checks 12570a68b904b02d0138bc5e78756dd0d10bad28in , thanks!